### PR TITLE
Fixes #5885

### LINKED ISSUE #5885: Editor button alignment
#### Description
For languages with longer strings, the editor buttons wrap a little awkwardly.  

For details & screenshots look at the initial bug report:
https://www.simplemachines.org/community/index.php?topic=567312.0

Note the bug report also has some recommended solutions.  

I am recording this here for input.  It is pretty easy to  reproduce this using Spanish, especially if you use a lower res monitor, or smaller browser window.  

I have not tested the proposed solution.  My only concern is whether it scales responsively properly.  The current solution may look awkward with windows of certain sizes, but it works & scales as you shrink & grow the browser window.  

### Environment (complete as necessary)
- Version/Git revision: RC2 & current
- Database Type: na
- Database Version: na
- PHP Version: na

### PR DESCRIPTION
**Pull request by XinYenFon**
Fixes #5885

edit, added the issue number for reference.

### CODE REVIEW

File: Themes/default/css/index.css [hunk 1]
```diff
@@ -3499,11 +3499,26 @@ ul.post_options li {
...
+	justify-content: space-between;
+	width: 100%;
+}
 #post_confirm_buttons, #shortcuts {
 	padding: 12px 0;
 }
 #post_confirm_buttons .smalltext {
-	padding: 0 10px;
+	flex: 1;
+	word-break: break-all;
```
Comment (SychO9) [inline Themes/default/css/index.css]: I think `break-word` would be better, makes it easy to read the sentence.

File: Themes/default/css/index.css [hunk 2]
```diff
@@ -3499,11 +3499,26 @@ ul.post_options li {
 #post_draft_options .settings strong {
 	color: #555;
 }
+#post_confirm_buttons {
+	display: flex;
+	flex-direction: row;
+	flex-wrap: wrap;
+	justify-content: space-between;
+	width: 100%;
```
Comment (SychO9) [inline Themes/default/css/index.css:3507]: would be cleaner to remove this and then drop the `class="floatright"` from the `post_confirm_buttons` element I believe.

**Comment by XinYenFon** [comment]
Did the requested changes.

**Review by SychO9 — APPROVED** [review]
nice thanks

**Review by jdarwood007 — APPROVED** [review]

**Pull request merged**